spec/fixtures/invalid-typechecked/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-unused-variable": true
```

Comment:
I was thinking of adding a rule that the current documentation lists as needing type-checking. But I'm not strongly opposed to leaving it as is if you wish.